dlib_face_detection/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(dlib_face_detection)
+
+find_package(catkin REQUIRED COMPONENTS
+  cob_people_detection
+  cob_perception_msgs
+  cob_vision_utils
+  cv_bridge
+  dlib_lib
```

Comment:
oh i see ... the folder is called `dlib` but the package in it `dlib_lib`. 
I will change the package name into `dlib` too